ci-operator/step-registry/cloud-bulldozer/e2e-benchmarking/rds-core/cloud-bulldozer-e2e-benchmarking-rds-core-commands.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+pushd /tmp
+
+
+git clone https://github.com/vishnuchalla/e2e-benchmarking --branch v0.0.1 --depth 1
```

Comment:
This is quickly test this functionality. Once done we will add checks to only run on e2e PRs.

Reply:
Could we have this step moved to **_ci-operator/step-registry/openshift-qe/rds-core_** instead please?
Also, you may want to assign yourself to **[PERFSCALE-3646](https://issues.redhat.com/browse/PERFSCALE-3646) - Create a RDS Prow step**

Reply:
Thanks for sharing the JIRA. I have hyperlinked in the one I have in this PR description.

Reply:
> Could we have this step moved to ci-operator/step-registry/openshift-qe/rds-core instead please?

Unfortunately no. This test has a different purpose to only run against e2e CI. And we want these e2e CI tests to be decoupled from the actual production CI runs to avoid unnecessary noise.

For example if we look at cd-v2, we have two versions of it.
One is the actual production run: https://github.com/openshift/release/tree/master/ci-operator/step-registry/openshift-qe/cluster-density-v2 
Another that runs againt e2e CI with minimal configuration: https://github.com/openshift/release/tree/master/ci-operator/step-registry/cloud-bulldozer/e2e-benchmarking/cluster-density-v2
Both are different and have a different purpose.